source/operations/checklists/software.rst

```diff
@@ -28,10 +28,19 @@ MinIO Pre-requisites
        Check with your operating system's documentation for how to synchronize time with a time server.
 
    * - :octicon:`circle`
-     - Disable system services that index, scan, or audit the filesystem, system-level calls, or kernel-level calls (``mlocate``, ``auditd``). 
+     - Disable system services that index, scan, or audit the filesystem, system-level calls, or kernel-level calls
        These services can reduce performance due to resource contention or interception of MinIO operations.
 
-       You can alternatively configure these services to ignore or exclude MinIO directories and processes.
+       The following is a non-exhaustive list of services or softwares which should be disabled on hosts running MinIO:
```

Comment:
```suggestion
       MinIO strongly recommends disabling at least the following services on hosts running MinIO:
```

Reply:
:thinking: 'at least' implies there is more, which is true. And now that I Think of it either way we're inviting the question. maybe:

```suggestion
      MinIO strongly recommends uninstalling or disabling the following services on hosts running MinIO: